apache/superset
# feat: fallback to external password store for sqlalchemy connections

### SUMMARY
Superset can be configured to use an external store for database passwords. This is useful if you a running a custom secret distribution framework and do not wish to store secrets in Superset’s meta database.
This useful capability can be extended by providing  Superset’s meta database as a fallback in the case the administration doesn't want to use an external store for a particular database 

### TESTING INSTRUCTIONS
Set `SQLALCHEMY_CUSTOM_PASSWORD_STORE`   to point to an external lookup function. and the user does not have to check where her password is saved. Current tests must continue to work

 ### ADDITIONAL INFORMATION
